packages/app/index.html

```diff
@@ -4,7 +4,8 @@
   <meta charset="UTF-8">
   <link rel="icon" href="/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Vite App</title>
+  <link rel="stylesheet" href="./basicStyles.css">
```

Comment:
Remove this line

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317